workflow/snakemake_rules/core.smk

```diff
@@ -44,7 +44,7 @@ rule filter:
         metadata = build_dir + "/{build_name}/metadata.tsv",
         log = build_dir + "/{build_name}/filtered.log"
     params:
-        group_by = "clade lineage",
+        group_by = config.get("groupby", "clade lineage"),
         sequences_per_group = config['sequences_per_group'],
         min_date = config['min_date'],
         min_length = config['min_length'],
```

Comment:
Being nit-picky here. Would be nice to make config key == param key, to be (mostly) consistent with what's below.

```suggestion
        group_by = config.get("group_by", "clade lineage"),
        sequences_per_group = config['sequences_per_group'],
        min_date = config['min_date'],
        min_length = config['min_length'],
```

Reply:
makes sense. Thanks.